app/templates/view_times.html

```diff
@@ -16,10 +16,7 @@
         }
     }
 
-    var oldBody = document.getElementsByTagName('tbody')[0];
```

Comment:
Looks like the rebase accidentally nuked this bit of code. This code should replace `form.submit()`.